### DISCUSSION TITLE
Is there Ubuntu package available for latest version in 22.04?

### DISCUSSION THREAD

**Discussion by noorul** [edenhill/librdkafka#3941] [Q&A]
The official Ubuntu 22.04 repository has a lower version 1.8.0-1build1. Is there any other repository from which we can get the latest version to be installed on 22.04? If not when will this be available?

**Comment by OneCricketeer** [comment]
> Is there any other repository from which we can get the latest version 

Github? And compile manually? 😉

**Comment by noorul** [comment]
Unofficial one that I created https://launchpad.net/~noorul/+archive/ubuntu/librdkafka
